crates/binstalk-downloader/Cargo.toml

```diff
@@ -61,7 +61,7 @@ tokio = { version = "1.35.0", features = [
 tokio-tar = "0.3.0"
 tokio-util = { version = "0.7.8", features = ["io"] }
 tracing = "0.1.39"
-hickory-resolver = { version = "0.25.0-alpha.3", optional = true, features = [
+hickory-resolver = { version = "0.25.0-alpha.5", optional = true, features = [
```

Comment:
we should pin it to avoid future breakage

Reply:
Yeah that's good idea, we will pin it until a 0.25.0 has come out that guarantees stable API

Reply:
Thanks updated